README.md

```diff
@@ -71,13 +71,20 @@ For further information see:
 
 ## Installation
 
-The python script `repoload.py` is self contained.  Just drop the file
-in a folder that your environment variable `PATH` references.
+The installation of `repoload` is possible over PyPI or by directly using the
+python script.
+
+PyPI:
+
+    $ python3 -m pip install repoload
```

Comment:
I think it's used because then it's more clear which python version is used to execute pip.
`python -m pip` is also used in the https://docs.python.org/3/installing/index.html 

Based on this stackoverflow answer https://stackoverflow.com/a/25749976 there is no difference between the two.

> --user

I will add the `--user` flag